api/app/src/Repository/StagingSelectedCandidateRepository.php

```diff
@@ -21,10 +21,11 @@ public function __construct(ManagerRegistry $registry)
     /**
      * @return StagingSelectedCandidate[]
      */
-    public function getDistinctCandidates(): iterable
+    public function getDistinctCandidates(): array
```

Comment:
This will be fixed in the next ticket, to yield candidates one at a time into the builder section of the ingest.